policy-service/src/main/java/uk/gov/gchq/palisade/service/policy/config/ApplicationConfiguration.java

```diff
@@ -31,24 +30,25 @@
 
 import uk.gov.gchq.palisade.jsonserialisation.JSONSerialiser;
 import uk.gov.gchq.palisade.service.policy.exception.ApplicationAsyncExceptionHandler;
+import uk.gov.gchq.palisade.service.policy.model.AuditErrorMessage;
+import uk.gov.gchq.palisade.service.policy.service.ErrorHandlingService;
 import uk.gov.gchq.palisade.service.policy.service.NullPolicyService;
 import uk.gov.gchq.palisade.service.policy.service.PolicyService;
+import uk.gov.gchq.palisade.service.policy.service.PolicyServiceAsyncProxy;
 import uk.gov.gchq.palisade.service.policy.service.PolicyServiceCachingProxy;
 import uk.gov.gchq.palisade.service.policy.service.PolicyServiceHierarchyProxy;
 
-import java.util.Optional;
 import java.util.concurrent.Executor;
 
 /**
  * Bean configuration and dependency injection graph
  */
 @Configuration
-@EnableAutoConfiguration
 public class ApplicationConfiguration implements AsyncConfigurer {
```

Comment:
See data-service (i think?), prefer to put the AsyncConfigurer stuff in a separate class